src/header.rs

```diff
@@ -131,6 +133,9 @@ pub struct GnuExtSparseHeader {
     pub padding: [u8; 7],
 }
 
+/// Mask to select only file type and the executable bit.
+const DETERMINISTIC_FILE_MODE_MASK: u32 = libc::S_IFMT as u32 | libc::S_IXUSR as u32 | libc::S_IXGRP as u32 | libc::S_IXOTH as u32;
```

Comment:
I personally always find these a little wonky to read, but the octal notation should work well here, right?